components/BCRegistriesAccount.vue

```diff
@@ -209,7 +242,15 @@ export default class BCRegistriesAccount extends Vue {
     return !!token
   }
 
-  get showNewFlag (): boolean {
+  get showNewFlagWills (): boolean {
+    return getFeatureFlag('bcregistry-ui-wills-new-chip')
```

Comment:
You should cast these to `boolean` since they come back from LaunchDarkly as `any`.